### PR TITLE
doc: add support status page - v3

### PR DESCRIPTION
**Pull request by jasonish**
Convert the support status wiki page to a chapter in our user guide.

Previous PR: https://github.com/OISF/suricata/pull/8944
Changes from previous PR:
- Incorporate all comments
- Add Debian 12
- Remove many names from maintainer if OISF member, or now supported by the OISF.
- Vendor tier.

Published to RTD:
https://jasonish-suricata.readthedocs.io/en/doc-support-status-v3/support-status.html#

### CODE REVIEW

**Comment by catenacyber** [comment]
CI looks a bit red :-/

File: doc/userguide/support-status.rst [hunk 1]
```diff
@@ -0,0 +1,462 @@
...
+   :width: 100%
+
+   +------------------+----------+----------+-------+--------------------------------+
+   |Distribution      |Version   |Support   |QA     |Notes                           |
+   +==================+==========+==========+=======+================================+
+   |CentOS            |Stream    |OISF      |       |                                |
+   +------------------+----------+----------+-------+--------------------------------+
+   |Fedora            |Active    |OISF      |       |                                |
+   +------------------+----------+----------+-------+--------------------------------+
+   |FreeBSD           |11        |OISF      |       |                                |
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:119]: looks like FreeBSD 11 is EOL

File: doc/userguide/support-status.rst [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+.. table::
+   :width: 100%
+
+   +--------------------+-------------------------+----+---------------+
+   |Capture Type        |Maintainer               |QA  |Notes          |
+   +====================+=========================+====+===============+
+   |PF_RING             |OISF                     |    |               |
+   +--------------------+-------------------------+----+---------------+
+   |NETMAP (Linux)      |OISF                     |    |               |
+   +--------------------+-------------------------+----+---------------+
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:218]: lets add dpdk here at 2

File: doc/userguide/support-status.rst [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+   +--------------------+--------------------------+----+---------------+
+   |Capture Type        |Maintainer                |QA  |Notes          |
+   +====================+==========================+====+===============+
+   |NFLOG               |Community / Giuseppe Longo|    |               |
+   +--------------------+--------------------------+----+---------------+
+   |Endace/DAG          |Community                 |    |               |
+   +--------------------+--------------------------+----+---------------+
+   |AF_XDP              |Community                 |    |               |
+   +--------------------+--------------------------+----+---------------+
+   |AF_PACKET (eBPF/XDP)|Eric Leblond              |    |               |
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:235]: should probably move to 2

File: doc/userguide/support-status.rst [hunk 4]
```diff
@@ -0,0 +1,462 @@
...
+
+.. table::
+   :width: 100%
+
+   +--------------------+--------------------------+----+---------------+
+   |Capture Type        |Maintainer                |QA  |Notes          |
+   +====================+==========================+====+===============+
+   |NFLOG               |Community / Giuseppe Longo|    |               |
+   +--------------------+--------------------------+----+---------------+
+   |Endace/DAG          |Community                 |    |               |
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:231]: unmaintained?

File: doc/userguide/support-status.rst [hunk 5]
```diff
@@ -0,0 +1,462 @@
...
+.. table::
+   :width: 100%
+
+   +---------------+-------------------------+----+---------------+
+   |Capture Type   |Maintainer               |QA  |Notes          |
+   +===============+=========================+====+===============+
+   |IPFW           |                         |    |               |
+   +---------------+-------------------------+----+---------------+
+
+Detection
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:262]: remove section

File: doc/userguide/support-status.rst [hunk 6]
```diff
@@ -0,0 +1,462 @@
...
+.. table::
+   :width: 100%
+
+   +---------------+----------------+----+----------------+
+   |Detect         |Maintainer      |QA  |Notes           |
+   +===============+================+====+================+
+   |GeoIP          |                |    |                |
+   +---------------+----------------+----+----------------+
+
+Outputs
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:308]: remove section

File: doc/userguide/support-status.rst [hunk 7]
```diff
@@ -0,0 +1,462 @@
...
+   |Type        |Maintainer                          |QA  |Notes                  |
+   +============+====================================+====+=======================+
+   |http.log    |OISF                                |    |                       |
+   +------------+------------------------------------+----+-----------------------+
+   |tls.log     |OISF                                |    |scheduled for removal  |
+   +------------+------------------------------------+----+-----------------------+
+   |pcap-log    |OISF                                |    |                       |
+   +------------+------------------------------------+----+-----------------------+
+
+AppLayer Protocols
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:347]: remove section

File: doc/userguide/support-status.rst [hunk 8]
```diff
@@ -0,0 +1,462 @@
...
+
+   +------------+-------------------+----+-------------------------+
+   |Protocol    |Maintainer         |QA  |Notes                    |
+   +============+===================+====+=========================+
+   |modbus      |                   |    |                         |
+   +------------+-------------------+----+-------------------------+
+   |enip/cip    |                   |    |                         |
+   +------------+-------------------+----+-------------------------+
+
+Operation modes
```
Comment (victorjulien) [inline doc/userguide/support-status.rst:396]: keep

**Comment by jasonish** [comment]
Continued at https://github.com/OISF/suricata/pull/9177, with CI fix.

**Pull request closed without merging**